src/utils/index.ts

```diff
@@ -5,7 +5,8 @@ import { Plugins } from '@capacitor/core';
 import { toastController } from '@ionic/vue';
 import { saveAs } from 'file-saver';
 import { DateTime } from 'luxon';
-import Papa from 'papaparse'
+import Papa from 'papaparse';
+import Encoding from 'encoding-japanese';
```

Comment:
Add this package in package.json file and also add types for the same `types/encoding-japanese`.